warehouse/templates/manage/project/publishing.html

```diff
@@ -118,6 +125,7 @@
           {{ field_errors(github_publisher_form.environment) }}
         </div>
       </div>
+      {{ prefilled_warning() if prefilled_provider == "github" }}
```

Comment:
Nitpick: IMO putting this at the top of the form instead of the bottom might catch the user's attention slightly better/flow a little better 🙂

Reply:
(Same for the others)

Reply:
Fixed! I also updated the screenshot